src/resource_management.ts

```diff
@@ -0,0 +1,57 @@
+import { AbstractCursor, ChangeStream } from './beta';
+import { MongoClient } from './mongo_client';
+import { ClientSession } from './sessions';
+
+/**
+ * @public
+ * @experimental
```

Comment:
No.  beta and public are mutually exclusive.  But we need to export this, otherwise API extractor complains because public classes implement this interface.